README.md

```diff
@@ -42,13 +51,19 @@ Generate mariaDB Docker Container and Create Database Instance for CWS:
 docker run -d -p 3306:3306 -e MYSQL_DATABASE=__DB_NAME__ -e MYSQL_ROOT_PASSWORD=__ROOT_PW__ -e TZ=America/Los_Angeles --name mdb103 mariadb:10.3
 ```
 
+Replace `__DB_NAME__` with your desired database name. <br />
+Replace `__ROOT_PW__` with your desired password.
+
 `__DB_NAME__` and `__ROOT_PW__` must match parameters set in script file: `<personal-dev>.sh`
 
 Directly access mariaDB with:
 
 ```
 mysql -h 127.0.0.1 -u root -p
 ```
+Enter the password above when prompted.
+
+_CWS must have been built prior to directly accessing mariaDB with the MySQL monitor_
```

Comment:
Can you clarify this a little more?  Not sure I understand what you mean here.

Reply:
Added context to the statement. Intention was to prevent someone from trying to directly access mariaDB immediately after running docker container using MySQL

Reply:
@RonnyFrayRegato 
I think you could clean this up a little. Maybe re-write it:

_In order for CWS to establish a connection during the installation process, the database must be accessible with the given DB host, username, and password. The development build script will use the above DB configuration. See the **Building CWS** section for an example build script._